### DISCUSSION TITLE
收不到激活码，技术支持邮件无法回复

### DISCUSSION THREAD

**Discussion by emeraldbast** [getlantern/lantern-forum-cn#730] [Q&A] [answered]
emeraldbast@163.com 收不到激活码 请求帮助
<img width="638" alt="receipt 1" src="https://github.com/user-attachments/assets/f550e595-f139-4c68-a521-5cb268853960">
<img width="581" alt="receipt 2" src="https://github.com/user-attachments/assets/5fe96411-f784-49c1-a5e5-b8921421d391">

**Comment by emeraldbast** [comment]
@Mike777Khoo 请问可以看下吗

**Reply by MaxMKMax** [reply]
@emeraldbast 您好. 好的没问题我会帮您提交问题给技术团队. 但要等到周三了. 我们在做维修. 不好意思让您久等了!

**Reply by emeraldbast** [reply]
记得处理下哦

**Reply by MaxMKMax** [reply]
@emeraldbast 
好的. 已经帮您提交问题上去了. 今天还弄不了激活码. 我明天再帮您跟进. 不好意思让您久等了.

**Reply by emeraldbast** [reply]
跟进了吗？什么时候能好呢？不能用的时间有补偿吗？不能的话能按比例退款吗？

**Reply by MaxMKMax** [reply]
@emeraldbast 您好. 我已经帮您跟进了. 等待回复. 我帮您问问看可以帮您加时间吗. 谢谢您的耐心等待.

**Comment by greySkiesMarsh** [comment, marked as answer]
Hello, 

This is Marsh. I've gone ahead and activated one year Pro status on this account, expiration is set to 7/30/2025. 

Thanks for your support!

**Reply by emeraldbast** [reply]
@Mike777Khoo 抱歉重新核实了，目前问题已经解决，感谢帮助
